Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.11 as builder
+WORKDIR /rathena
+RUN apk add --no-cache cmake make gcc g++ gdb zlib-dev mariadb-dev linux-headers valgrind bash
```

Comment:
i don't think you need cmake, gdb, or valgrind

Reply:
I'll test without them and update the commit, I've just put these because they are in `tools/docker/Dockerfile`

Reply:
removed cmake, valgrind, gdb and bash